spec/integration/manifest.spec.yml

```
@@ -410,6 +410,21 @@ spec:
                 - access
           deployment_modes:
             $ref: "#/definitions/deployment_modes"
+          links:
+            description: List of links related to policy template.
+            type: array
+            items:
+              type: object
+              properties:
```

Probably it's interesting to add here `additionalProperties` to false, so just these entries are the ones to be allowed

```suggestion
              type: object
              additionalProperties: false
              properties:
```